tests/system/provisioning/basic_cluster/roles/master-role/tasks/main.yml

```
@@ -100,3 +108,18 @@
 
 - name: Start Wazuh
   command: /var/ossec/bin/wazuh-control restart
+
+- name: "Install necessary dependencies"
+  command: /var/ossec/framework/python/bin/python3.9 -m pip install lockfile filetype certifi testinfra
+
+- name: "Create a file to call the socket"
+  copy:
+    dest: "/tmp/send_msg.py"
+    content: |
+      import sys
+      sys.path.append('/wazuh-qa/deps/wazuh_testing')
+      from wazuh_testing import wazuh_db
+
+      result = wazuh_db.query_wdb(f'global sql select * from agent where id={sys.argv[1]}')
```

Maybe is better to change this script in order to receive the full query as an argument. In this way, this script could be more useful for the rest of the tests of the environment.


```suggestion
      result = wazuh_db.query_wdb(f'{sys.argv[1]}')
```